docs/docs/overview/quickstart/get-started-with-ruby.md

```diff
@@ -0,0 +1,303 @@
+
+:::info
+Please do not hardcode your credentials in a file in production. Use environment variables instead.
+:::
+
+To test your API key, call the methods on the client to interact with the Novu API and display the result on your ruby console:
+
+```ruby
+puts client.notifications
+```
+
+Run the file `index.rb`:
+```bash
+ruby index.rb
+```
+
+You should see a result as below:
+```json
+{
+  "page": 0,
+  "totalCount": 0,
+  "pageSize": 10,
+  "data": []
+}
+```
+
+## Set Up A Channel Provider
+
+A channel provider is a service that provides one or more notification functionality such as sending an email, SMS, push notification etc. Our [integration store](https://web.novu.co/integrations) includes four channels: Email, SMS, Chat, and Push. These channels have multiple providers associated with them.
+
+| Channel | Providers                                                           |
+| ------- | ------------------------------------------------------------------- |
+| Email   | MailGun, Mandrill, MailJet, Amazon SES, Sendgrid, Postmark, Netcore |
+| SMS     | Twilio, Amazon SNS, Plivo, SMS, SMSCentral, Kannel, Infobip, Termii |
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
+
+## Create A Notification Template
+
+A notification template is the blueprint for the notifications that will be sent. It holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
+
+The template includes the following:
+
+- Notification template name and Identifier
+- Channel tailored content:
+
+| Channel | Content Style                                                                                 |
+| ------- | --------------------------------------------------------------------------------------------- |
+| Email   | 1. Custom Code (HTML) with option to use custom variables via the handlebars , {{ }}, syntax. |
+|         | 2. Click and place UI items with the visual template editor.                                  |
+| SMS     | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| Chat    | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| In-App  | Text                                                                                          |
+
+:::info
+Proper authorization needs to be set for the Chat channel for subscribers.
+:::
+
+Please proceed to create a notification template.
+
+1. Click **Workflows** on the left sidebar of your Novu dashboard. 
+TODO: (Add a picture that shows the sidebar)
```

Comment:
When are you doing to add the TODO:?

Reply:
I attached the picture to the description of this PR. 

I noticed that the images are served from cloudinary. i was hoping you help with the upload and update the part with the correct image url

Reply:
Oh okay. This is the URL: https://res.cloudinary.com/dxc6bnman/image/upload/v1688745999/guides/workflows_m60hkc.png

Reply:
done